Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Spawnable/PrefabDocument.cpp

```diff
@@ -158,4 +153,23 @@ namespace AzToolsFramework::Prefab::PrefabConversionUtils
             return false;
         }
     }
+
+    bool PrefabDocument::RefreshPrefabDom() const
```

Comment:
Currently we just discard the return value for this, and the failure path triggers an assert - it may not be worth keeping the return value around - this otherwise is a nice refactor that makes things clearer

Reply:
It has indeed lost its purpose so replaced the `bool` with a `void`. It can always be added back in again if needed.